pyppeteer/page.py

```diff
@@ -550,8 +566,7 @@ def _onCertificateError(self, event: Any) -> None:
                 'cookies': items,
             })
 
-    async def addScriptTag(self, options: Dict = None, **kwargs: str
-                           ) -> ElementHandle:
+    async def addScriptTag(self, **kwargs: str) -> ElementHandle:
```

Comment:
Might as well just expand this, although this somewhat depends on `domworld.py`. You could just defer it for #30
```suggestion
    async def addScriptTag(self, url: str = None, path: str = None, content: str = None, _type: str = None) -> ElementHandle:
```